OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -2454,95 +2427,64 @@ private static long getLastSessionTime() {
    }
 
    /**
-    * Setting to control how OneSignal notifications will be shown when one is received while your app
-    * is in focus.
-    * <br/><br/>
-    * {@link OneSignal.OSInFocusDisplayOption#Notification Notification} - native notification display while user has app in focus (can be distracting).
-    * <br/>
-    * {@link OneSignal.OSInFocusDisplayOption#InAppAlert In-App Alert (Default)} - native alert dialog display, which can be helpful during development.
-    * <br/>
-    * {@link OneSignal.OSInFocusDisplayOption#None None} - notification is silent.
-    *
-    * @param displayOption the {@link OneSignal.OSInFocusDisplayOption OSInFocusDisplayOption} to set
+    * You can call this method with {@code false} to opt users out of receiving all notifications through
+    * OneSignal. You can pass {@code true} later to opt users back into notifications.
+    * @param disable whether to subscribe the user to notifications or not
```

Comment:
this comment is wrong

Reply:
ah, logic was flipped with the `setSubscription` to `dsiablePush` rename but this comment was missed.